files/image_config/syslog/host_umount.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#This script is invoked at the closure of syslog socket during reboot
+#This will stop journal services, unmount /var/log and delete loop device 
+#associated to /host to ensure proper unmount of /host
+
+journal_stop() {
+    systemctl stop systemd-journald.service
+    systemctl stop systemd-journald.socket
+    systemctl stop systemd-journald-audit.socket
+    systemctl stop systemd-journald-dev-log.socket
+}
+
+delete_loop_device() {
+    umount /var/log
+    if [[ $? -ne 0 ]]
+    then
+	exit 0
```

Comment:
>	 [](start = 0, length = 1)

Do not mix space and tab. We prefer space. Could you also reformat the whole file?

Reply:
Accepted